mantidproject/mantid
# Stop script tab order changing when Workbench is closed and reopened

**Description of work.**
This PR fixes a bug that caused the order of tabs in the script editor to change when Workbench is closed and reopened.

Report to: Duc Le

**To test:**
1. In Workbench, open multiple script files, remember the order that the tabs are in, and close Workbench.
2. Reopen Workbench and check that the tabs are in the same order.
3. Reorder the tabs and check that they remain in that order after closing and reopening Workbench.

Fixes #27074 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
